cfgov/unprocessed/css/organisms/featured-content-module.less

```diff
@@ -0,0 +1,104 @@
+// Duplicated from current implementation in Capital Framework v4.6.0
+// with minor edits (to be released in CFv4.7.0)
+// This can be deleted safely when this project is ported to CFv4
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  // Fix the doubled border issue
```

Comment:
Could reference any relevant GH issue in the comment that explains this behavior, if there is one.